media/CircleGraph/index.js

```diff
@@ -52,6 +52,7 @@ const viewMode = {
   viewer: 0, // default circle viewer
   selector: 1, // circle partition editor node selector
   visq: 2, // quantization error viewer
+  visqselr: 3, //quantization error viewer whicj is able to select nodes
```

Comment:
typo whicj

Reply:
why is this mode added?
what is `visqselr`? this word is hard to read.

Reply:
> why is this mode added?

new mode is intended to provide .visq colormap, as well as `selector` capabilities to provide a user handy selection of nodes.

> what is `visqselr`? this word is hard to read.

Yes. The name is hard to read, i'll gladly change it with any other more fortunate name.